src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -44,7 +46,7 @@ public Datastore() {
   /** Stores the Message in Datastore. */
   public void storeMessage(Message message) {
     Entity messageEntity = new Entity("Message", message.getId().toString());
-    messageEntity.setProperty("user", message.getUser());
+    messageEntity.setProperty("user", message.getUser().toString());
```

Comment:
why did you do this?

Reply:
I did not do this...I'm guessing Travis did this or it was an accident.

Reply:
Can you remove? It implies getUser does not return a string, which is confusing.